packages/docs/content/5_components/alert.mdx

```diff
@@ -6,12 +6,16 @@ Alerts keep users informed of important and sometimes time-sensitive changes.
 
 ## `<Alert>`
 
+<StorybookExample componentName="alert" sourceFilePath="/components/Alert/Alert.tsx" />
```

Comment:
I just noticed that this turned into `https://github.com/CMSgov/design-system/blob/master/packages/design-system/src//components/Alert/Alert.tsx` with `...src//components...`

Reply:
Fixed!